zephyr/lib/cpu.c

```diff
@@ -129,6 +132,7 @@ int cpu_enable_core(int id)
 		k_smp_cpu_start(id, secondary_init, NULL);
 		return 0;
 	}
+#endif /* CONFIG_PM */
 
 	k_smp_cpu_resume(id, secondary_init, NULL, true, false);
```

Comment:
interesting, I understand, that this PR mostly aims to fix compilation with `CONFIG_PM=n`, but logically, the call to `k_smp_cpu_resume()` seems even more PM-related to me than `k_smp_cpu_start()`. Should we move `#endif` below it?

Reply:
done